.github/workflows/doc-build.yml

```diff
@@ -45,13 +45,7 @@ jobs:
     - name: install-pip
       run: |
         pip3 install setuptools wheel
-        pip3 install -r scripts/requirements-base.txt
         pip3 install -r scripts/requirements-doc.txt
-        pip3 install west==0.10.1
```

Comment:
NAK. We need west installed in the doc build workflow to get west API docs, and we need to control the exact version so the content of west-apis.rst matches.

Reply:
Then why does the doc build not fail without west, how's that possible?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/master/doc/conf.py#L127

Reply:
Ah OK: the doc build supports both with or without west. Then shouldn't CI try both?

Reply:
Removed that bit